app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'application_record'
+
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  # A method that updates the comments counter for a post.
+  def update_comments_counter_in_post
+    counter = post.comments_counter
+    counter ? post.update(comments_counter: 1) : post.update(comments_counter: counter + 1)
+  end
+end
```

Comment:
- [ ] [OPTIONAL] This approach works well, however it would be nice to use `after_save` Rails callback to call the `update_comments_counter_in_post` method to increase your `comments_counter` each time a new comment is created 👍 
### Hint:
**In order to do this, you'd have to make your `update_comments_counter_in_post` method private**
So you'd have something like this 👇 
```
after_save :update_comments_counter_in_post

private

def update_comments_counter_in_post
  post.increment!(:comments_counter) # this line is a shorter way of incrementing your comments_counter
end
```

Reply:
Thanks, @codepantha. I'll read about this in the documentation and implement it.